docs/playbooks/README.md

```diff
@@ -0,0 +1,87 @@
+# Playbooks
+
+This folder contains playbooks for handling common incidents and operational procedures for the 121 Platform. Playbooks are step-by-step guides that assist team members in diagnosing and resolving specific types of issues efficiently and consistently.
+
+## Purpose
+
+- **Consistency**: Ensure that incidents are handled in a uniform manner.
+- **Efficiency**: Reduce the time to resolution by providing clear instructions.
+- **Knowledge Sharing**: Capture expertise and make it accessible to all team members.
+- **Training**: Serve as a resource for onboarding new team members.
+
+## How to Use
+
+When an incident occurs, refer to the relevant playbook in this folder to guide your response. If a playbook does not exist for the incident you're handling, consider creating one after the issue is resolved.
+
+## Creating a New Playbook
```

Comment:
> So maybe we could create a separate private repo to manage these playbook. In general this may also be a good idea for the techdoc we now maintain in teams (and is very annoying to navigate).

I do not have a problem navigating Tech Doc, so curious to understand better what is annoying.

I do believe that the doc on Teams is quite accessible as that is where we communicate internally. IMO creating a separate private repo makes it less accessible.

> As I really struggle finding old bugs in devops

I do not have this problem. Maybe I am better at searching, and/or need to find bugs less frequently.

Reply:
> I do not have a problem navigating Tech Doc, so curious to understand better what is annoying.
- Ctrl + shit + f is much less advanced in teams than it is in VsCode
- It loads slow 
- No PR review process in teams
- No versioning

> I do not have this problem. Maybe I am better at searching, and/or need to find bugs less frequently.
'Really struggle' is maybe over the top. But I do find it a lot easier to find stuff in a our code base than in Azure Devops

Reply:
I'm resolving this for now. We can discuss when we meet.